contrib/depends/Makefile

```diff
@@ -53,7 +53,10 @@ endif
 host_arch=$(firstword $(subst -, ,$(canonical_host)))
 host_vendor=$(word 2,$(subst -, ,$(canonical_host)))
 full_host_os:=$(subst $(host_arch)-$(host_vendor)-,,$(canonical_host))
+host_os:=$(findstring android,$(full_host_os))
```

Comment:
Since the host triple is of the form <CPU>-linux-android, we need to detect android before checking for regular linux...